### DISCUSSION TITLE
no files found

### DISCUSSION THREAD

**Discussion by yesasdasds** [WilliamVenner/gmpublisher#211] [Help]
whenever i try upload a map i get this error that says "no files found" i dont know why it cant see the files i did everything correctly but the same GOD DAMN error showed up
![Screenshot 2024-02-15 173707](https://github.com/WilliamVenner/gmpublisher/assets/126644287/fcaf465c-b3ea-4c03-acb3-afe55e2c3cc3)

what can i do to fix this? as it seems like noone else has had this problem

**Comment by WilliamVenner** [comment]
https://wiki.facepunch.com/gmod/Workshop_Addon_Creation#preparingyouraddon
